base/cvd/cuttlefish/host/libs/web/android_build_api.cpp

```diff
@@ -232,8 +236,13 @@ Result<std::string> BuildApi::BuildStatus(const DeviceBuild& build) {
   std::string url = api_base_url_ + "/builds/" +
                     http_client->UrlEscape(build.id) + "/" +
                     http_client->UrlEscape(build.target);
+  bool firstQueryAdded = false;
```

Comment:
Sorry I just noticed that cvd uses snake_case to name variables. Let's use `first_query_added`.

Reply:
Fixed, thanks for the reminder!